transports/sql/native-publish-subscribe.md

```diff
@@ -0,0 +1,42 @@
+---
+title: SQL Server Native Publish Subscribe
+summary: Describes the native publish subscribe implementation in the SQL Server transport
+reviewed: 2019-11-08
+component: SqlTransport
+versions: '[5,)'
+---
+
+The SQL Server transport implements the publish-subscribe pattern. On version 4 and below, this feature relies on message-driven pub-sub which requires a separate persistence for storage of subscription information. On version 5 and above, the transport handles subscription information natively and a separate persistence is not required.
```

Comment:
I think most references in the doco use "In version", not "On version" . I am not an English expert. Is there a subtle difference here?

Reply:
You are correct. I will fix it

Reply:
The endpoint runs _on_ a version of the transport. The features exist _in_ a version of the transport.